lib/fog/vsphere/requests/compute/create_vm.rb

```diff
@@ -22,7 +22,7 @@ def create_vm(attributes = {})
           vm_cfg[:cpuHotAddEnabled] = attributes[:cpuHotAddEnabled] if attributes.key?(:cpuHotAddEnabled)
           vm_cfg[:memoryHotAddEnabled] = attributes[:memoryHotAddEnabled] if attributes.key?(:memoryHotAddEnabled)
           vm_cfg[:firmware] = attributes[:firmware] if attributes.key?(:firmware)
-          vm_cfg[:bootOptions] = boot_options(attributes, vm_cfg) if attributes.key?(:boot_order) || attributes.key?(:boot_retry)
+          vm_cfg[:bootOptions] = boot_options(attributes, vm_cfg)
```

Comment:
What was the reason we are dropping the if statement here for the boot order?

Reply:
The `boot_options` method will not be called if `boot_order` or `boot_retry` are not present, but we need this method to be called for Secure Boot configuration. Also,  `boot_order` is always passed by default, so the condition is unnecessary.